### ISSUE TITLE
The tutorial bubbles keep showing on after having pressed 'Done'

### ISSUE THREAD

**Issue by Wtower** [AllMyChanges/original.allmychanges.com#31]
![allmychanges com 2015-05-22 12 11 48](https://cloud.githubusercontent.com/assets/10957886/7767721/79f2d086-007d-11e5-9365-12d3ca4c55f9.png)

**Comment by svetlyak40wt** [comment]
Yes this is slightly annoying. I'm thinking about mechanism which will keep track which tooltips you already seen. But this requires a third-party library modification. I'm using a great [intro.js](http://usablica.github.io/intro.js/) module from Usabilica.
